### ISSUE TITLE
Remove points from the polygon

### ISSUE THREAD

**Issue by sirKitKat** [arthur-e/Wicket#141]
There is currently no way to remove points from the polygon

**Comment by arthur-e** [comment]
Hi @sirKitKat, could you provide more information? Are you talking about the online sandbox? Wicket is intended for translating between WKT and mapping framework objects; I think that editing geometry is outside of the scope of this library.

**Comment by sirKitKat** [comment]
When using the tool: http://arthur-e.github.io/Wicket/sandbox-gmaps3.html
You can draw or edit lines/shapes, you can move points and add points by draging the middle points of a line segment but there is no way to remove points from an existing polygon.

**Comment by arthur-e** [comment]
Yes, the tool you linked is just a demonstration of the Wicket library. If someone has a Pull Request to improve these demos, I welcome it, but they don't make up any part of the functionality of the Wicket library.
